src/server/graphql/types/User.js

```diff
@@ -111,6 +121,31 @@ const User = new GraphQLObjectType({
         return (userId === source.id) ? source.tms : undefined;
       }
     },
+    teams: {
```

Comment:
At this point, I'm confused by when we decide to handle field resolution in `server/graphql/queries` modules vs. in `server/graphql/types` modules.

Reply:
Type vs. Field. let's chat it up if it's still a little foggy, once you get it down it'll make GraphQL seem a lot cooler!